README.md

```diff
@@ -0,0 +1,77 @@
+# ITKMorphologicalContourInterpolation
+
+![Build, test, package](https://github.com/KitwareMedical/ITKMorphologicalContourInterpolation/workflows/Build,%20test,%20package/badge.svg)
+
+[![PyPI](https://img.shields.io/pypi/v/itk-morphologicalcontourinterpolation.svg)](https://pypi.python.org/pypi/itk-morphologicalcontourinterpolation)
+
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/KitwareMedical/ITKMorphologicalContourInterpolation/blob/master/LICENSE)
+
+## Overview
+
+An [ITK](https://itk.org)-based implementation of morphological contour interpolation based off the paper:
+
+Albu AB1, Beugeling T, Laurendeau D.
+"A morphology-based approach for interslice interpolation of anatomical slices from volumetric images."
+IEEE Trans Biomed Eng.
+2008 Aug;55(8):2022-38.
+doi: 10.1109/TBME.2008.921158.
+
+👨‍💻 [`Live API Demo`] ✨
```

Comment:
This works!

We should enable output image compression by default. Label images are highly compressible, so the savings are significant.

Reply:
Tracking this here: https://github.com/InsightSoftwareConsortium/itk-wasm/issues/1064